Travel Scheduler/Views/GoToMoreOptionsCell.m

```diff
@@ -32,27 +41,39 @@ - (void)initWithType:(NSString *)type
     if(self.imageView == nil) {
     self.imageView = [[UIImageView alloc] init];
     }
+    
+    self.titleLabel = [[UILabel alloc] initWithFrame:CGRectMake(0,10,self.contentView.bounds.size.width ,self.contentView.frame.size.height/5)];
+    instantiateTitle(self.titleLabel);
+    [self.contentView addSubview:self.titleLabel];
+    
+    
     instantiateImageView(self.imageView);
+    self.imageView.frame = CGRectMake(self.contentView.frame.size.width/4,self.titleLabel.frame.origin.y  + self.titleLabel.frame.size.height + 10,self.contentView.bounds.size.width/2,self.contentView.bounds.size.height/2);
```

Comment:
You should probably put this line into the layoutsubviews method so it autolayouts.

Reply:
Done!